src/index.js

```diff
@@ -184,6 +184,10 @@ function setupAudioOnPageLoad() {
         recording.play()
       })
     })
+    .catch(e=>{
+      // fixme (eddie): now what?
+      console.log(e)
+    })
```

Comment:
I added the catch because the console was showing annoying errors for many words that do not have recordings

Reply:
Fair enough!